src/plugins/typography.js

```diff
@@ -8,13 +8,11 @@ module.exports = (fontSizes) =>
   });
 
```

Comment:
A comment explaining why certain properties are set to `initial` would be helpful, I think. Future readers will spend a second wondering before assuming.

Also, not a big deal, but why `initial` instead of the desired value? The effect is the same. But setting, say, `text-rendering` to `auto` is a more explicit and saves the reader a moment's time.